tools/nxstyle.c

```diff
@@ -50,6 +50,45 @@
 
 #define LINE_SIZE    512
 
+#define FATAL(m)       message(FATAL, (m), 0, 0)
+#define FATALFL(m,s)   message(FATAL, (m), -1, -1)
+#define WARN(m, l, o)  message(WARN, (m), (l), (0))
+#define ERROR(m, l, o) message(ERROR, (m), (l), (0))
+#define INFO(m, l, o)  message(INFO, (m), (l), (0))
+
+/****************************************************************************
+ * Private types
+ ****************************************************************************/
+
+enum class_e {
+  INFO,
+  WARN,
+  ERROR,
+  FATAL
+};
+
+const char *class_text[] =
+{
+  "info",
+  "warning",
+  "error",
+  "fatal"
+};
+
+enum file_e {
+  unknown,
+  c_header,
+  c_source,
```

Comment:
Enumeration values are always in upper case.

Reply:
There should be no dangling comma on the final value of the enumeration

Reply:
Good catch.  GCC is tolerant of dangling , at the end of enumerations, but other compilers are not.  Still not proper programming style in either case.